code/modules/reagents/newchem/paradise_pop.dm

```diff
@@ -0,0 +1,66 @@
+
+/*
+	Paradise Pop reagents
+	Created through the bottler machine via bottler_recipes, not through standard reactions
+	Eventually will have special effects (minor mostly) tied to their reagents, but for now are purely for flavor
+
+	Make sure to yell at me to finish giving them effects later
+	-FalseIncarnate
+*/
+
+
+//Paradise Punch: No effect, aside from maybe messages about how tasty it is or something
+/datum/reagent/paradise_punch
+	name = "Paradise Punch"
+	id = "paradise_punch"
+	description = "Tastes just how you'd think Paradise would if you could bottle it."
```

Comment:
@tigercat2000 Paradise can infact be a proper noun. https://en.wiktionary.org/wiki/Paradise#Proper_noun It's not just the server.